log4j2/default/log4j2.xml

```diff
@@ -16,6 +16,8 @@
     <Logger name="org.apache.sshd.common.util.SecurityUtils" level="warn"/>
     <Logger name="org.apache.tomcat.util.net.NioSelectorPool" level="warn" />
     <Logger name="org.hibernate.validator.internal.util.Version" level="warn" />
+    <Logger name="org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver" level="error" />
+    <Logger name="org.apache.catalina.core.ContainerBase" level="off" />
```

Comment:
really, turned `off` completely ???

Reply:
Unfortunately, this gets logged by JUL with level 'severe' (https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html). I am still working on a different solution, but this gets the job done :-/

Reply:
See https://github.com/Oreste-Luci/apache-tomcat-8.0.26-src/blob/05aeff976f6a75346c95cf6d3a9c8add8b9cb0bc/java/org/apache/catalina/core/StandardWrapperValve.java#L256 (Responsible for logging the underlying ApiTokenQuotaExceeded exception).